packages/daimo-api/src/server/router.ts

```diff
@@ -103,7 +104,12 @@ export function createRouter(
     return opts.next();
   });
 
+  const sentryMiddleware = trpcT.middleware(
+    Sentry.Handlers.trpcMiddleware({ attachRpcInput: true }) as any
```

Comment:
The `any` type here is tied to this issue: https://github.com/getsentry/sentry-javascript/issues/9536